### ISSUE TITLE
Skeleton lee94 can not work

### ISSUE THREAD

**Issue by LucasBoTang** [scikit-image/scikit-image#4168]
## Description
Skeleton lee94 can not work

## Way to reproduce
```python
scribble = skeletonize(worst_area, method='lee')
```

```python
# your output here
skeletonize() got an unexpected keyword argument 'method'
```

**Comment by jni** [comment]
@LucasBoTang it looks like you found the documentation for our *development* version, which has some more cutting edge features than the released version (0.15), including this one (which was added in #3904). To use Lee's method, use `skeletonize3d` until scikit-image 0.16 is released (should be in the next couple of weeks!). After that, the code above should work.

Thanks for the report, and I hope this helps! Closing for now but feel free to continue asking questions if something is not clear.

**Comment by LucasBoTang** [comment]
Thank you very much!
